docs/source/getting-started/statistical-modeling/regression.ipynb

```diff
@@ -277,7 +277,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": null,
```

Comment:
It looks like this notebook wasn't reevaluated after changes. (Could imagine a test that every notebook has every cell evaluated in order.) I think CI will tell us if there are errors, but it wouldn't notices changes in output short of that.